Makefile

```diff
@@ -63,22 +81,16 @@ prere:
 	$(PREFIX)cd libft && make re
 
 clean:
-	$(PREFIX)rm -f $(OBJS) $(ENDPOINT_OBJ) $(VANIA_ENDPOINT_OBJ) $(TANIA_ENDPOINT_OBJ)
-	$(PREFIX)rm -f $(OBJS) $(ENDPOINT_OBJ) $(VANIA_ENDPOINT_OBJ) $(TANIA_ENDPOINT_OBJ)
+	$(PREFIX)rm -f $(OBJS) $(ENDPOINT_OBJ) $(TANIA_ENDPOINT_OBJ) $(TEST_TOOL_OBJS)
+	$(PREFIX)@if [ -d $(OBJ_F) ]; then $(PREFIX)rm -rf $(OBJ_F); fi
 
 fclean: clean
-	$(PREFIX)rm -f $(NAME)
-	$(PREFIX)rm -f $(ENDPOINT_OBJ)
+	$(PREFIX)rm -f $(NAME) $(TEST_TOOL_FNAME)
 	$(PREFIX)rm -f $(ENDPOINT_OBJ)
+	$(PREFIX)@if [ "$(wildcard build)" ]; then $(PREFIX)rm -r build; fi
```

Comment:
This should have happened in clean already, in the dependency

Reply:
please view it in view file, its already in this commit

https://github.com/ioannnikulin/minishell/blob/0714c6f8f613229f1a5e48c82030299d433a6546/Makefile